bazel/linux/test.bzl

```diff
@@ -20,9 +97,13 @@ def qemu_test(name, kernel_image, run, qemu_binary = None,
         config: dict, all additional attributes to pass to the
             kernel_qemu_run rule, generally created with mconfig().
     """
+
+    runner_script = mcreate_rule(name, test_runner, "test-runner", [],
```

Comment:
update the docstring of the function: now this macro instantiates 3 rules: {name}-test-runner, {name}-run, and {name}

Reply:
Fixed